src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ruleNode.tsx

```diff
@@ -110,11 +125,31 @@ class RuleNode extends React.Component<Props> {
     );
   };
 
+  getMailActionFields = (_: string, __: FormField) => {
+    const {data} = this.props;
+    const isInitialized =
+      data?.targetType !== undefined && `${data.targetType}`.length > 0;
+    if (!isInitialized) {
+      const newData: IssueAlertRuleAction = {...(data as IssueAlertRuleAction)};
+      newData.targetType = MailActionTargetType.IssueOwners;
+      this.handleMailActionChange(newData);
```

Comment:
Hmm I don't think we want to do this, I believe this is causing a side effect during the rendering phase.

Reply:
Deleting this chunk of code. Turns out the parent component is already handling supplying of initial value.

Reply:
Thanks for the catch though, I agree that we don't want this.